### ISSUE TITLE
query: multiple 'query.replica-label' flag

### ISSUE THREAD

**Issue by amalucelli** [thanos-io/thanos#1571]
**Thanos, Prometheus and Golang version used**:
```sh
thanos, version 0.7.0 (branch: HEAD, revision: c6eaf68bec32aefd280318eaef6121d6ddb20d10)
  build user:       circleci@f199bdee6152
  build date:       20190902-15:43:19
  go version:       go1.12.5
```

**What happened**

I got this error message when I tried to use multiple flags on the `query` component:

> flag 'query.replica-label' cannot be repeated

**What you expected to happen**

As implemented on this PR (https://github.com/thanos-io/thanos/pull/1362), I was expecting to be able to use multiple flags :) maybe I'm missing something.

I even saw on the CHANGELOG this topic described:

> #1362 query.replica-label configuration can be provided more than once for multiple deduplication labels like: --query.replica-label=prometheus_replica --query.replica-label=service.

I also followed the latest documentation that instruct on how to use it.

**How to reproduce it (as minimally and precisely as possible)**:

```sh
docker run --rm thanosio/thanos:v0.7.0 query --query.replica-label=prometheus_replica --query.replica-label=prometheus_region

Error parsing commandline arguments: flag 'query.replica-label' cannot be repeated
usage: thanos query [<flags>]

```

**Comment by brancz** [comment]
cc @krasi-georgiev

**Comment by yeya24** [comment]
Are you using release v0.7.0? #1362 is not included in it. If you want to use this feature, you can build from master.

**Comment by amalucelli** [comment]
@yeya24 yeah, maybe I got confused, idk.. I saw this change on the CHANGELOG under v0.7.0, that's why

**Comment by jojohappy** [comment]
Hi, maybe it is our fault. #1362 is merged after v0.7.0 released. 

Will fix CHANGELOG soon. Sorry for that.
